tests/integration/test_executable_user_defined_function/test.py

```diff
@@ -228,3 +228,15 @@ def test_executable_function_sum_json_python(started_cluster):
     )
 
     node.query("DROP TABLE test_table;")
+
+def test_executable_function_input_nullable_python(started_cluster):
+    skip_test_msan(node)
```

Comment:
Why do we skip msan for all tests? Just because of speed?

Reply:
It has problems with `vfork` system call.